src/Sign.Cli/Resources.resx

```diff
@@ -170,7 +170,7 @@
     <value>RFC 3161 timestamp server URL.</value>
   </data>
   <data name="VerbosityOptionDescription" xml:space="preserve">
-    <value>Sets the verbosity level. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].</value>
+    <value>Sets the verbosity level. Allowed values are None, Critical, Error, Warning, Information, Debug, and Trace.</value>
```

Comment:
Why are values starting with upper case? Also why not offering abbreviation? like ` -n | none`

```suggestion
<value>Sets the verbosity level. Allowed values are 'none', 'critical', 'error', 'warning', 'information', 'debug', and 'trace'.</value>
```

Reply:
I made them lower case to be consistent with other option values (like `sha256`).  I also quoted other option values elsewhere (e.g.:  `sha256`).

The original verbosity option description, which I'm updating in this PR, I stole from dotnet.exe.  However, I didn't get around to implementing it that way.  Instead, I used logging as defined in System.CommandLine.  This is why the option values changed.  It is also why abbreviations are not offered, because it was never a feature to begin with, just an incorrect description.  We don't use option value abbreviations elsewhere, and after chatting with @clairernovotny on this, it's a new feature request.